docker/sandbox-lite/flyte-entrypoint-default.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/sh
```

Comment:
Last time we deprecated the non dind images because they are not used by anyone, Any reason of building it again ?

Reply:
I didn't know it was deprecated. If so, I'm going to remove it.